java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -1323,6 +1325,290 @@ public static CompletableFuture<Object> toggle(
                 client, new ArgsBuilder().add(gs(JSON_TOGGLE)).add(key).add(path).toArray());
     }
 
+    /**
+     * Appends the specified <code>value</code> to the string stored at the specified <code>path
+     * </code> within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @param value The value to append to the string. Must be wrapped with single quotes. For
+     *     example, to append "foo", pass '"foo"'.
+     * @param path The path within the JSON document.
+     * @return
+     *     <ul>
+     *       <li>For JSONPath (<code>path</code> starts with <code>$</code>):<br>
+     *           Returns a list of integer replies for every possible path, indicating the length of
+     *           the resulting string after appending <code>value</code>, or <code>null</code> for
+     *           JSON values matching the path that are not string.<br>
+     *           If <code>key</code> doesn't exist, an error is raised.
+     *       <li>For legacy path (<code>path</code> doesn't start with <code>$</code>):<br>
+     *           Returns the length of the resulting string after appending <code>value</code> to the
+     *           string at <code>path</code>.<br>
+     *           If multiple paths match, the length of the last updated string is returned.<br>
+     *           If the JSON value at <code>path</code> is not a string of if <code>path</code>
+     *           doesn't exist, an error is raised.<br>
+     *           If <code>key</code> doesn't exist, an error is raised.
+     *     </ul>
+     *
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", "$", "{\"a\":\"foo\", \"nested\": {\"a\": \"hello\"}, \"nested2\": {\"a\": 31}}").get();
+     * var res = Json.strappend(client, "doc", "baz", "$..a").get();
+     * assert Arrays.equals((Object[]) res, new Object[] {6L, 8L, null}); // The new length of the string values at path '$..a' in the key stored at `doc` after the append operation.
```

Comment:
Nit: replace \`\` with \<code\>\<\/code\> in Javadoc

Reply:
It is ok to keep `` ` `` in example comments